.azure-devops/common/setup_steps.yml

```diff
@@ -0,0 +1,10 @@
+steps:
+    # install nodejs
+    - task: NodeTool@0
+      inputs:
+        versionSpec: '16.x'
```

Comment:
Should the versionSpec not be `14.x`?

Reply:
I was running into an issue with `fs` or `fs-events` failing to install on our agents using a `v14.x` node version - switching to `v16` on the agents fixed it for me. Might have been related to how my package-lock.json was created (lots of stuff moved around when fixing vulns) - but I was using v16 locally... Is there an issue bumping this to 16 (LTS) or do we want to try keeping it on 14 if we can?

Here's the node version / support timeline if that helps us decide:
https://nodejs.org/en/about/releases/

Reply:
My concern is vscode `1.65.1` uses electron [`13.5.2`](https://github.com/microsoft/vscode/blob/1.65.1/package.json#L139) which depends on node [`14.16.0`](https://github.com/electron/electron/blob/v13.5.2/DEPS#L19). We need to at least test against the node runtime of the min version of vscode we support.

Reply:
Ok, I've rolled my local install to v14 and regenerated the package-lock file. Going to setup a pipeline run against this change and will commit it if everything looks good.